.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,53 @@
+<!--
+Thank you for creating this pull request!
+
+Provide a general summary of your changes in the Title above.
+And please provide enough information in the description below, so that others can review your pull request.
+-->
+
+## Pull Request Type
+
+- [ ] I have checked there is no other PR open for the same change.
+
+This is a:
+- [ ] Bug fix
+- [ ] New feature
+- [ ] QA update
+
+## Context
+<!--
+What do you want to achieve with this PR? Why did you write this code? What problem does this PR solve?
+What should be mentioned about this PR in the changelog?
+-->
+
+
+## Detailed Description
+
+
+
+
+
+## Quality assurance
+
+- [ ] This change does NOT contain a breaking change (fix or feature that would cause existing functionality to change).
+- [ ] I have commented my code, particularly in hard-to-understand areas.
+- [ ] I have added unit tests to accompany this PR.
+- [ ] The (new/existing) tests cover this PR 100%.
+- [ ] I have (manually) tested this code to the best of my abilities.
+- [ ] My code follows the style guidelines of this project.
+
+## Documentation
+
+For new features:
+- [ ] I have added a code example showing how to use this feature to the [`examples`](https://github.com/WordPress/Requests/tree/develop/examples) directory.
+- [ ] I have added documentation about this feature to the [`docs`](https://github.com/WordPress/Requests/tree/develop/docs) directory.
+    If the documentation is in a new markdown file, I have added a link to this new file to the Docs folder `README.md` file.
+
+<!--
+PLEASE MAKE SURE YOUR PR PASSES THE CI CHECKS.
```

Comment:
I personally don't like being yelled at. Also, it seems paradoxical to use "please" together with yelling.

I suggest avoiding all-caps text.
```suggestion
Please make sure your pull request passes all continuous integration checks!
```

Reply:
I've often found that with wordy PR templates, people don't read the comments, which is why I tried to make it stand out a little more, but I agree yelling is not a good thing.

I'll apply these changes, but will suggest adding some fencing inside the comment, so the text still stands out a bit more.